Sources/Core/State/LoadingMediaState.swift

```diff
@@ -162,7 +164,8 @@ final class LoadingMediaState: PlayerState {
         case .readyToPlay:
             guard let position = self.position else { moveToLoadedState(); return }
             let seekPosition = CMTime(seconds: position, preferredTimescale: context.config.preferedTimeScale)
-            context.player.seek(to: seekPosition) { completed in
+            context.player.seek(to: seekPosition) { [context] completed in
+                context.delegate?.playerContext(didCurrentTimeChange: context.currentTime)
                 guard completed else { return }
                 self.moveToLoadedState()
```

Comment:
weak self

Reply:
not needed here.
may be due to context property: unowned